Scala/Scala.sublime-syntax

```diff
@@ -1039,6 +1074,9 @@ contexts:
       scope: support.type.scala
       set: single-type-expression-tail
     - include: base-type-expression
+    - match: '(?=@)'
+      comment: the '@' indicates the beginning of an annotation
```

Comment:
With YAML defs we are moving away from `comment` keys and just writing a YAML comment.